lib/colorado_lottery.rb

```diff
@@ -0,0 +1,50 @@
+
+
+class ColoradoLottery
+  attr_reader :registered_contestants, :winners, :current_contestants
+
+  def initialize
+    @registered_contestants = {}
+    @winners = []
+    @current_contestants = {}
+  end
+
+  def find_game_interest(name, game_interest)
+    name.game_interests.find do |game|
+        if game_interest.name == game
+         return true
+       else
+         return false
+        end
+    end
+  end
+
+  def interested_and_18?(name, game_interest)
+    if name.age >= 18 || find_game_interest(name, game_interest) == true
```

Comment:
We want the contestant to be 18 AND interested, so you need to replace `||` with `&&`